.eslintrc.js

```diff
@@ -24,5 +23,6 @@ module.exports = {
     '@typescript-eslint/require-array-sort-compare': 'off',
     'no-lone-blocks': 'off',
     'no-useless-return': 'off',
+    '@typescript-eslint/member-delimiter-style': 'off',
```

Comment:
It was causing issues with semicolons in type/interface definitions. And its use is discouraged: https://typescript-eslint.io/rules/member-delimiter-style/